modules/s3-bucket/providers.tf

```diff
@@ -11,8 +12,9 @@ provider "aws" {
 }
 
 module "iam_roles" {
-  source  = "../account-map/modules/iam-roles"
-  context = module.this.context
+  source             = "../account-map/modules/iam-roles"
+  global_tenant_name = var.root_account_tenant_name
```

Comment:
This should not be needed. `iam-roles` should be configured with this information specifically so we do not need to pass it through all the root modules.
```suggestion
```

Reply:
To configure `iam-roles` with the root account tenant name, do we need to set the default value for `var.global_tenant_name` within `.../account-map/modules/iam-roles/variables.tf`? 
```
variable "global_tenant_name" {
  type        = string
  description = "Global tenant name"
  default     = "core" # this line
}
```
That does work, but I thought we would want to keep all variables values within components as non-specific. Is there another way to set that value that I'm missing?